roles/acme/tasks/challenge/dns-01/hetzner.yml

```diff
@@ -27,7 +27,7 @@
         headers:
           Auth-API-Token: "{{ acme_hetzner_auth_token }}"
       loop: "{{ acme_domain.subject_alt_name }}"
-      register: records
+      register: acme_records_result
```

Comment:
here seems also to be a surplus `_result`

Reply:
fixed